frontend/src/components/IssueSidebar.vue

```diff
@@ -35,6 +35,48 @@
         />
       </div>
 
+      <h2 class="textlabel flex items-center col-span-1 col-start-1">
+        {{ $t("common.when") }}<span v-if="create" class="text-red-600">*</span>
+        <div class="tooltip-wrapper">
+          <span class="tooltip w-60">{{
+            $t("task.earliest-allowed-time-hint")
+          }}</span>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            class="h-4 w-4"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path
+              stroke-linecap="round"
+              stroke-linejoin="round"
+              stroke-width="2"
+              d="M8.228 9c.549-1.165 2.03-2 3.772-2 2.21 0 4 1.343 4 3 0 1.4-1.278 2.575-3.006 2.907-.542.104-.994.54-.994 1.093m0 3h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z"
+            />
+          </svg>
```

Comment:
Is this SVG Icon in the Heroicons icon set? Use the Icon component instead. Check out #135 PR

Reply:
done